includes/functions-formatting.php

```diff
@@ -562,7 +565,7 @@ function yourls_esc_url( $url, $context = 'display', $protocols = array() ) {
  * @param string $url URL
  * @return string URL with lowercase scheme and protocol
  */
-function yourls_lowercase_scheme_domain( $url ) {
```

Comment:
High level of breaking change here. 😨
Is the function moved to compat somewhere?

Reply:
Good call. I'm adding it to `functions-deprecated.php`.